package.json

```diff
@@ -85,9 +91,21 @@
       ]
     },
     "angular-mock": "global:angular.mock",
-    "diff-match-patch": "diff_match_patch",
-    "dom-text-mapper": "DomTextMapper",
-    "dom-text-matcher": "DomTextMatcher",
+    "dom-anchor-fragment": "domAnchorFragment",
```

Comment:
Gross hackery. These are all UMD modules and we need to make sure they don't get tricked on pages that use AMD.